ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -470,29 +488,117 @@ class JITModBuilder {
       Instruction *CurrentInst = &CurrentBB->back();
       DominatorTree DT(*JITFunc);
       std::vector<Value *> LiveVals = getLiveVars(DT, CurrentInst);
+      // Naturally the current instruction is live too but wasn't included due
+      // to the way DominatorTree works.
+      LiveVals.push_back(CurrentInst);
+
+// Add arguments for stackmap pointer and size.
+#if defined(__x86_64)
```

Comment:
I think there's a cluster of attributes that cache platform specific types. Should we move this up next to them?

Reply:
Not in `jitmodbuilder.cc` it seems.

Reply:
Could you use [this](https://github.com/ykjit/yk/blob/41a38f74f60e68f28474e7e230ff1f52db6d13de/ykllvmwrap/src/jitmodbuilder.cc#L187) instead?

Reply:
Since this is used for `size_t`s we get from C, I think maybe `getIndexType` (https://llvm.org/doxygen/classllvm_1_1DataLayout.html#abbd9ed911d1d58b73f9571f300cc6a7b) is more appropriate, which is typically the same anyway.

Reply:
Ah, that doesn't return an `IntegerType` so this won't work. According to [1] we should be able to just use `IntPtrTy` for `size_t` as well.

Reply:
Fixed in 7799079